### DISCUSSION TITLE
多集群架构下higress能否跨集群监听ingress？

### DISCUSSION THREAD

**Discussion by zerodemoy** [alibaba/higress#419] [Q&A]
如题：类似istio的primary-remote架构，在primary集群的higress controller现在支持监听remote集群的ingress吗？

**Comment by johnlanni** [comment]
目前开源版本还不支持，可以看看企业版，有这个能力：https://www.aliyun.com/product/aliware/mse?spm=higress-website.topbar.0.0.0

**Reply by zerodemoy** [reply]
这个能力开源版本有支持的计划吗？

**Reply by johnlanni** [reply]
https://github.com/alibaba/higress/issues/425

建了issue收集需求，我们会根据社区用户的需求来排优先级
